QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+{
+	"@graph": [
+		{
+			"@id": "res:acqf",
+			"@type": "???:QualificationsFramework",
+			"ceasn:name": { "en": "African Continental Qualifications Framework" },
+			"ceterms:alternateName": { "en": "ACQF" },
+			"ceasn:description": { "en": "The ACQF is a comprehensive and inclusive meta-referencing qualifications framework, designed to support a holistic and systemic vision of learning, qualifications, and credentials. The ACQF is oriented to lifelong learning and supports parity of esteem of learning outcomes acquired in formal, non-formal and informal contexts, and all levels of education and training." },
+			"ceasn:dateValidFrom": "2022-11-01",
+			"ceterms:versionCode": "17",
+			"ceasn:source": [
+				"https://drive.google.com/file/d/1BnU5VOqS6hu2YRhowC7x6J1R1_p6UblV/view"
+			],
+			"ceasn:hasGlossary": [
+				"res:acqfGlossary"
+			],
+			"asn:hasProgressionModel": [
+				"res:acqfLevels"
+			]
+		},
```

Comment:
Add authors/contributors & publisher?

Reply:
Would we add everyone under "Authors and reviewers of this policy document" and/or "authors and reviewers of the ACQF guidelines..." or just one of those sets?  And, would we include contributors under "authors"?

I'm not quite sure what organization to attribute publishing to.

Reply:
I wouldn't sweat it too much at this stage, we just need to show that we can list those things. Presumably anyone publishing their own QF as data would know who wrote and published it.